compiler/rustc_llvm/llvm-wrapper/CoverageMappingWrapper.cpp

```diff
@@ -198,12 +229,17 @@ extern "C" void LLVMRustCoverageWriteFuncSectionNameToString(LLVMModuleRef M,
   WriteSectionNameToString(M, IPSK_covfun, Str);
 }
 
+extern "C" void LLVMRustCoverageWriteBitmapSectionNameToString(LLVMModuleRef M,
+                                                             RustStringRef Str) {
+  WriteSectionNameToString(M, IPSK_bitmap, Str);
+}
+
 extern "C" void LLVMRustCoverageWriteMappingVarNameToString(RustStringRef Str) {
   auto name = getCoverageMappingVarName();
   auto OS = RawRustStringOstream(Str);
   OS << name;
 }
 
 extern "C" uint32_t LLVMRustCoverageMappingVersion() {
-  return coverage::CovMapVersion::Version6;
+  return coverage::CovMapVersion::Version7;
 }
```

Comment:
This will be properly fixed by #121026.

(I'm a little surprised that #123409 doesn't touch this version number at all. Maybe the coverage mapping reader is not very picky about version numbers when reading MC/DC mappings.)

Reply:
i didn't know about this MR, and I was skeptical about hardcoding the number in this manner.
Looks like my doubts were indeed justified. Thanks !